daemon/prune.go

```diff
@@ -25,6 +25,27 @@ var (
 	// ErrPruneRunning is returned when a prune request is received while
 	// one is in progress
 	ErrPruneRunning = fmt.Errorf("a prune operation is already running")
+
+	containersAcceptedFilters = map[string]bool{
+		"label":  true,
+		"label!": true,
```

Comment:
Looked up the PR that implemented it, and indeed the format is `label!=key=val` or `label!=key`. Negative matches on _value_ alone don't seem to be supported (i.e., no `label=key!=val`), nor spaces in between (`label != key=val` etc.)

https://github.com/yongtang/docker/blob/702524732427ce028277f99f215e1fab297e6001/cli/command/utils.go#L95-L116